Parsers/ASimWebSession/Parsers/ASimWebSessionPaloAltoNetworks.yaml

```diff
@@ -50,49 +50,87 @@ ParserQuery: |
           and Activity == "THREAT"
           and DeviceEventClassID == "url"
       | parse AdditionalExtensions with URL ";PanOSActionFlags=" *
```

Comment:
As mentioned, I still don't get it. URL is there at the beginning without a key name? It might be a bug in CEF. Let's discuss in the meeting.

Reply:
Check